src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,70 @@
-export const Pagination = () => {};
+import React from 'react';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const itemsLengthArray = new Array(Math.ceil(total / perPage)).fill('item');
+  const onPageLinkClick = (value: number, condition = true) => {
+    return condition && onPageChange(value);
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={`${currentPage === 1 ? 'disabled ' : ''}page-item`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={
+            () => onPageLinkClick(currentPage - 1, currentPage !== 1)
+          }
+        >
+          «
+        </a>
+      </li>
+
+      {itemsLengthArray.map((_, i) => (
+        <li
+          // eslint-disable-next-line react/no-array-index-key
+          key={i}
```

Comment:
this rule is written for a reason, you should not disable it, the indexes change. The best option would be to create an object with generated stable identifiers for each number in the prepared array, or at least use the value, for you it is "i + 1"